_posts/2022-05-30-openmic-workflow-announcement

```diff
@@ -3,7 +3,7 @@ title: "Upcoming Open Mic - workflow and tooling for front end applications"
 categories: [Open Mic Announcement, Open Mic]
 ---
 
-The Open mic session starts on Friday 10-06-20202 at 10:30 at [this link](https://teams.microsoft.com/l/meetup-join/19:a53641a5c2b64fae9fed0985aa353847@thread.tacv2/1653900345165?context=%7B%22Tid%22:%22f345c406-5268-43b0-b19f-5862fa6833f8%22). Speaker Max Chopart her presentation with the topic workflow and tooling for front end applications.
+The Open mic session starts on Friday '10th June 2022' at 10:30 at [this link](https://teams.microsoft.com/l/meetup-join/19:a53641a5c2b64fae9fed0985aa353847@thread.tacv2/1653900345165?context=%7B%22Tid%22:%22f345c406-5268-43b0-b19f-5862fa6833f8%22). Speaker Max Chopart his presentation with the topic workflow and tooling for front end applications.
```

Comment:
ok, I appreciate change of his to her, but the sentence still does not make sense.

Reply:
hhh, yes. is it better now :)